lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+const getScriptFileName = require('../utilities/get-script-file-name');
+
+module.exports = function updateManifestFromHtml(root) {
+  const pkg = JSON.parse(fs.readFileSync(root + '/package.json', 'UTF8'));
+
+  // otherwise, we must parse the index.html file to figure out what's next
+  const indexHtml = fs.readFileSync(root + '/index.html', 'UTF8')
```

Comment:
IIRC, doesn't ember-cli provide an API the default index.html name and have the app call it something else? If that is the case, this will throw an error.